awx/api/serializers.py

```diff
@@ -4165,6 +4165,7 @@ def to_representation(self, obj):
                 else:
                     set_count += 1
             ret['stdout'] += u'\u001b[0m' * (set_count - reset_count)
+        ret['profiling_data'] = {}
```

Comment:
This isn't really accurate.  It would probably be better to `ret.pop('profiling_data').

Reply:
Agreed, updating